CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - Support FreeBSD as a representative for the UNIX platforms [#1054](https://github.com/eclipse-iceoryx/iceoryx/issues/1054)
 - Add event parameter to `findService` method [#415](https://github.com/eclipse-iceoryx/iceoryx/issues/415)
 - Implement stream operator for `ChunkReceiveResult` and `AllocationError` to be able to use it with ostream and LogStream [\#1062](https://github.com/eclipse-iceoryx/iceoryx/issues/1062)
+- Add `command_line.hpp` which contains a macro builder to parse command line arguments quickly and safeyly [#1067](https://github.com/eclipse-iceoryx/iceoryx/issues/1067)
```

Comment:
```suggestion
- Add `command_line.hpp` which contains a macro builder to parse command line arguments quickly and safely [#1067](https://github.com/eclipse-iceoryx/iceoryx/issues/1067)
```

Reply:
Please rebase to latest master and add this entry to the changelog for iceoryx v2.0